src/feeRouterDeployChildToParentRouter.ts

```diff
@@ -0,0 +1,142 @@
+import { Address, PublicClient, WalletClient, getAddress, pad, parseAbi } from 'viem';
+
+import { createTokenBridgeFetchTokenBridgeContracts } from './createTokenBridgeFetchTokenBridgeContracts';
+import { Prettify } from './types/utils';
+import { validateParentChain } from './types/ParentChain';
+import { WithTokenBridgeCreatorAddressOverride } from './types/createTokenBridgeTypes';
+import { childToParentRouter } from './contracts';
+
+/**
+ * This type is for the params of the feeRouterDeployChildToParentRouter function
+ */
+export type FeeRouterDeployChildToParentRouterParams = Prettify<
+  WithTokenBridgeCreatorAddressOverride<{
+    parentChainPublicClient: PublicClient;
+    orbitChainWalletClient: WalletClient;
+    parentChainTargetAddress: Address;
+    minDistributionInvervalSeconds?: bigint;
+    rollup?: Address;
+    parentChainTokenAddress?: Address;
+  }>
+>;
+
+// Default minimum distribution interval seconds
+const DEFAULT_MIN_DISTRIBUTION_INVERVAL_SECONDS = BigInt(60 * 60 * 24); // 1 day
```

Comment:
I suggest a default of 1 week instead (this is what we're using for Nova fee routing). The griefing vector is one outbox execution per interval (note that these fees will be paid by the AF/DAO, not the orbit chain owner)

Reply:
Oh, right. I forgot to ask about this. Changed. Thanks!